### ISSUE TITLE
Kian Ho: new volunteer via Mozilla Science Lab Collaborate

### ISSUE THREAD

**Issue by kianho** [vispy/vispy#736]
Hi! this project caught my attention as I've been consistently involved with developing visualisation tools in python throughout my phd (protein structure analysis). If you'd have me, I'd love to contribute to the python side of things, and it would give me a chance to learn more about some of the issues involved with WebGL/OpenGL development.

Thanks!
Kian<p><br><blockquote>This issue was created by @kianho via <a href='http://collaborate.mozillascience.org'>Mozilla Science Lab Collaborate</a></blockquote></p>

**Comment by rossant** [comment]
Hi, feel free to let us know what part of the project you'd be interested in contributing!

**Comment by boscoh** [comment]
Hi Kian, saw your message on the vispy list. Since you're into comp struct biology, you might be interested in a protein viewer that I've prototyped in vispy: https://github.com/boscoh/pyball. Also, my phd was on beta-sheets. I too am in Melbourne and know Mike Kuiper.

**Comment by kianho** [comment]
hello! I'm primarily interested in the python side of things but after looking through the source I'll need to spend some time familiarising myself with vispy and openGL. I did some basic opengl back in undergrad but I haven't touched it since.

@boscoh yes! Mike was one of my phd supervisors and I have encountered some of your work before, I didn't realise your were involved with this project. Beta-sheets...I've had nightmares after spending years looking at them.
